### PR TITLE
fix(model): Use global step to update learning rate

### PR DESCRIPTION
**Pull request by boss-chanon**
## Why this PR
Update learning rate by ```global_step```

## Changes
- Update learning rate by ```global_step```

## Related Issues
Close #

## Checklist
- [ ] PR should be in the [Naming convention](../../docs/PR_NAMING.md)
- [ ] Assign yourself in to Assigneees
- [ ] Tag related issues
- [ ] Constants name should be ALL_CAPITAL, function name should be snake_case, and class name should be CamelCase
- [ ] complex function/algorithm should have [Docstring](https://peps.python.org/pep-0257/)
- [ ] 1 PR should not have more than 200 lines changes (Exception for test files). If more than that please open multiple PRs
- [ ] At least PR reviewer must come from the task's team (model, eval, data)

### CODE REVIEW

File: src/model/openthaigpt_pretraining_model/lightning/utils.py [hunk 1]
```diff
@@ -223,13 +223,18 @@ def val_step(self):
     def train(self):
         self.opt.zero_grad()
         for epoch in range(self.start_epochs, self.epochs):
+            if epoch < self.start_epochs:
```
Comment (boat1603) [inline src/model/openthaigpt_pretraining_model/lightning/utils.py:226]: I think `epoch` has been skipped by `for epoch in range(self.start_epochs, self.epochs)`.

Comment (boss-chanon) [reply src/model/openthaigpt_pretraining_model/lightning/utils.py]: Oh, I just saw.

Comment (boss-chanon) [reply src/model/openthaigpt_pretraining_model/lightning/utils.py]: Now,I already remove it.

**Review by boat1603 — APPROVED** [review]

**Review by MoosaTae — APPROVED** [review]
